### PR TITLE
Ot153 34 organization edit form

### PR DESCRIPTION
**Pull request by federicorojas11**
## JIRA
[OT153-34](https://alkemy-labs.atlassian.net/jira/software/c/projects/OT153/boards/226?modal=detail&selectedIssue=OT153-34)

## Short description

> Added organization edit component with form validations
> The submit button only validates the fields, as it says on the ticket requirements.
> Validation for required fields fires on submit, and for optional social media links fires on changes 

## Screenshot or Video

https://user-images.githubusercontent.com/47337947/155770714-317ad471-fc86-4d78-b5c0-8d5de23a5c3d.mp4

## Teammate collaborator

- [ ] Someone from the team collaborated (**_if so, please tag them after this line_**).

### Sanity Check Steps

- [x] All console logs have been removed.
- [x] This change was tested on my localhost.
- [ ] This change depends for another ticket.
- [x] All linked JIRA acceptance criteria are satisfied.
- [ ] Does this change require environment variables.

### CODE REVIEW

File: src/app/core/enums/regex.enum.ts [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+  TWITTER_REGEX = "https://twitter.com/[A-Za-z0-9 - _  . ]{4,}$",
 }
 
 export const EMAIL_REGEX: RegExp = new RegExp(regex.EMAIL_REGEX);
 export const SYMBOLS_REGEX: RegExp = new RegExp(regex.SYMBOLS_REGEX);
 export const DIGIT_REGEX: RegExp = new RegExp(regex.DIGIT_REGEX);
 export const WORD_REGEX: RegExp = new RegExp(regex.WORD_REGEX);
-export const ALLDIGIT_REGEX: RegExp = new RegExp(regex.ALLDIGITS_REGEX);
\ No newline at end of file
+export const ALLDIGIT_REGEX: RegExp = new RegExp(regex.ALLDIGITS_REGEX);
```
Comment (RicardoMaldonado93) [inline src/app/core/enums/regex.enum.ts:16]: Good idea! :clap:

File: src/app/core/util/validators/form.validators.ts [hunk 2]
```diff
@@ -1,19 +1,37 @@
...
-    return patternValidator( EMAIL_REGEX, { hasEmail: true });
+  return patternValidator(EMAIL_REGEX, { hasEmail: true });
 }
 export function allDigitValidator(): ValidatorFn {
-    return patternValidator( ALLDIGIT_REGEX, { hasAllNumber: true });
-}
\ No newline at end of file
+  return patternValidator(ALLDIGIT_REGEX, { hasAllNumber: true });
+}
+export function facebookLinkValidator(): ValidatorFn {
```
Comment (RicardoMaldonado93) [inline src/app/core/util/validators/form.validators.ts:29]: Good idea! :clap:

File: src/app/features/app-routing.module.ts [hunk 3]
```diff
@@ -43,6 +45,10 @@ const routes: Routes = [
         path: "organization",
         component: OrganizationDetailsComponent,
       },
+      {
```
Comment (RicardoMaldonado93) [inline src/app/features/app-routing.module.ts:48]: mover al backoffice, debido a que es una ruta de edicion

**Review by RicardoMaldonado93 — COMMENTED** [review]
Please fix the observations to continues with the PR

File: src/app/features/app-routing.module.ts (hunk 3 above)
Comment (federicorojas11) [reply src/app/features/app-routing.module.ts:48]: Lo acabo de revisar, está en la ruta 'backoffice/organization/editar', y componente en carpeta backoffice... Avisame si es correcto xfas , hago rebase igualmente

**Pull request merged**